lib/instrumentation/openai.js

```diff
@@ -131,6 +132,17 @@ module.exports = function initialize(agent, openai, moduleName, shim) {
               index
             }).serialize()
 
+            // Track information for the feedback message API.
+            const tracker = segment.transaction.llm.conversation_id_tracker
```

Comment:
might be good to extract this code into its own function, thoughts?

Reply:
Where should that function live? Is it a method of `tracker`?

Reply:
Chatting on Zoom, we will leave this as-is for now and come back with a "clean up" pass.